bin/yaml/v.yaml

```diff
@@ -0,0 +1,11 @@
+compilers:
+  v:
+    type: ziparchive
+    dir: v-{name}
+    url: https://github.com/vlang/v/releases/download/weekly.{name}/v_linux.zip
+    folder: v
+    check_exe: v --version
+    after_stage_script:
+      - export VMODULES=$HOME/.vmodules
```

Comment:
This won't be persistent. The installed system cannot see or write to `$HOME` - this kind of work around will need to be done in the code that runs the compiler in CE itself.

Reply:
You'll have to override https://github.com/compiler-explorer/compiler-explorer/blob/main/lib/base-compiler.ts#L432 in your compiler class, and set `env['VMODULES']` to that value. You can get the path with `path.join(path.dirname(this.compiler.exe), '../.vmodules')`, I think

Reply:
Thanks, I've pushed a commit to my main CE PR